buildSrc/src/main/java/org/opensearch/gradle/testclusters/OpenSearchCluster.java

```diff
@@ -367,11 +370,15 @@ private void commonNodeConfig() {
         } else {
             nodeNames = nodes.stream().map(OpenSearchNode::getName).map(this::safeName).collect(Collectors.joining(","));
         }
+        String httpProtocol = "http";
+        if (nodes.stream().map(OpenSearchNode::getHttpProtocol).anyMatch(protocol -> protocol == "https")) {
+            httpProtocol = "https";
+        }
```

Comment:
If I set the protocol to 'foobar' it will be ignored, I think you just want to read the `OpenSearchNode::getHttpProtocol` value and use that.

Reply:
That makes sense. I will swap over.